clinica/pydra/t1_volume/create_dartel/__init__.py

```diff
@@ -0,0 +1 @@
+from . import cli
```

Comment:
I'm wondering whether we should homogenize the naming of the cli files... Some pydra pipelines (like t1-linear or pet-volume use the connection `name_of_the_pipeline_cli.py`, while some use just `cli.py`).